src/components/view/ListView.vue

```diff
@@ -439,7 +439,7 @@ export default {
         value: this.editableValue
       }).then(json => {
         this.editableValueKey = null
-
+        this.$store.dispatch('RefreshFeatures')
```

Comment:
@Pearl1594 why are we doing this here? Assuming we do list capabilities fetching already on log in?

Reply:
@rhtyd If this isn't done, then unless one logs out and logs back in or does a hard refresh, the store isn't going to be updated with the new value of 'kubernetesclusterexperimentalfeaturesenabled' in the capabilities list